src/index.html

```diff
@@ -20,43 +20,6 @@ <h2 class="pitch-title">Specifying IPFS and the Interplanetary stack.</h2>
   <p class="pitch-subtitle">
     The technology that powers the content-addressable web is being standardized here.
   </p>
-  <!--
-  <div class="basic-grid">
-    <div class="pitch">
-      <h3>High-quality open standards</h3>
-      <p>
-        We aim to fully define implementation behaviour and to empower anyone to build their
-        own system that fits into the interplanetary ecosystem without needing to reverse-engineer
-        the behavior of other implementations.
-      </p>
-    </div>
-    <div class="pitch">
-      <h3>Comprehensive test suites</h3>
-      <p>
-        A specification is only as useful as it is both testable and tested. We are building extensive
-        test suites that allow specifications to prove their interoperability and implementations to demonstrate
-        their correctness.
-        </p>
-    </div>
-    <div class="pitch">
-      <h3>Testing infrastructure</h3>
-      <p>
-        We build tools to support detecting interoperability issues early and often as part of CI/CD pipelines
-        and to report on implementation maturity as the stack evolves so that people who build on top of the
-        interplanetary stack can know what they can use in production.
-      </p>
-    </div>
-    <div class="pitch">
-      <h3>An open governance model</h3>
-      <p>
-        We guarantee community ownership of the standards, of the standards-production process, and of
-        the infrastructure to operate it, in collective pursuit of efficient, transformative, ethical
-        technology.
-      </p>
-    </div>
-  </div>
-  -->
-
```

Comment:
:thought_balloon: just flagging we are removing this, but this content may be useful later.

Reply:
I can re-add it